push_test/pushclient.py

```diff
@@ -0,0 +1,335 @@
+import asyncio
+import base64
+import json
+import uuid
+import sys
+import traceback
+
+import aiohttp
+import websockets
+
+
+class PushException(Exception):
+    pass
+
+
+def log_msg(out=sys.stdout, **msg):
+    print(json.dumps(msg), file=out)
+
+
+class PushClient(object):
```

Comment:
In Python 3, the (object) bit can be left off entirely as all classes now inherit from object by default.